packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramRenderer.tsx

```diff
@@ -219,6 +215,7 @@ export const DiagramRenderer = ({ diagramRefreshedEventPayload, selection, setSe
       snapToGrid={snapToGrid}
       snapGrid={[GRID_STEP, GRID_STEP]}
       connectionMode={ConnectionMode.Loose}
+      autoPanOnConnect={false}
```

Comment:
I did not find a way to dynamically change this value. This value default to true an allow the user to move the viewport when connecting. The issue is that it should be false when the pop up menu used to choice the type of connection is open and true otherwise.

Reply:
I put autoPanOnConnect back to true.